source/proof_address/lib/proof_address.rb

```diff
@@ -3,6 +3,8 @@
 require 'faraday'
 require 'retries'
 
+require_relative './ssm_helper' if !defined?(IdentityIdpFunctions::SsmHelper)
```

Comment:
this was my guess at "let's hope that the ssm_helper gets copied in to be next to this file"